README.md

```diff
@@ -33,7 +42,8 @@ And add to the top of your project's RuboCop configuration file:
   ~~~
 
 For more information about inheriting configuration from a gem please check
-[RuboCop's documentation](https://rubocop.readthedocs.io/en/latest/configuration/#inheriting-configuration-from-a-dependency-gem).
+[RuboCop's
+documentation](https://rubocop.readthedocs.io/en/latest/configuration/#inheriting-configuration-from-a-dependency-gem).
```

Comment:
I think you should update this link as it points to the old docs site.

Reply:
Indeed, thank you! 🙇🏼